streaming/base/storage/download.py

```diff
@@ -96,6 +96,11 @@ def _download_file(unsigned: bool = False,
         if obj.netloc in requester_pays_buckets:
             extra_args['RequestPayer'] = 'requester'
 
+    # Check for canned ACL environment variable
```

Comment:
I think canned acls are only for uploading?

Reply:
Canned ACLs can be used for both uploading and downloading objects in Amazon S3. When you upload an object to S3, you can specify a canned ACL to control who can access the object. For example, you can use the public-read canned ACL to make the object publicly readable, or you can use the private canned ACL to make the object only accessible to the owner of the bucket.

Canned ACLs can also be used to control who can download objects from S3. When you download an object from S3, the requester's permissions are checked against the object's ACL to determine whether they are allowed to download the object. For example, if an object has the public-read canned ACL, then anyone can download the object. However, if an object has the private canned ACL, then only the owner of the bucket can download the object.

Reply:
Right, I'm pretty sure what that says is "when uploading, you can specify a canned acl to apply to the object you are uploading. when downloading the requester's permissions will be checked against the acl that the _uploader_ specified". but i dont think the downloader needs to specify an acl.

Reply:
You are right! I verified that we don't need it during downloading. It failed with this exception `ValueError: Invalid extra_args key 'ACL', must be one of: ChecksumMode, VersionId, SSECustomerAlgorithm, SSECustomerKey, SSECustomerKeyMD5, RequestPayer, ExpectedBucketOwner` if I pass `ACL`. Will update the PR to just keep it for upload. Thanks for your review.